docs/docs/Releases/release220.md

```diff
@@ -0,0 +1,67 @@
+# Release 2.2.0
+
+Date: 5th of April, 2023
+
+## Overview
+
+Klaw version 2.2.0 is a minor release with several new features and
+improvements. Key highlights of this release include redesigned My
+Team's Requests page, pre-validation of schema requests, sensitive ACL
+data is now masked unless you are the owner and unified Kafka & Schema
+cluster administration.
+
+## What's new in Klaw 2.2.0
+
+Redesigned My Team's Request screens with React UI
+- With the switch to React, this release features a
+redesign of the following user interfaces:
+
+-   **My Team's Topic Requests**: Users can now view all of their
+    team's topic requests using the new UI.
+-   **My Team's Subscription Requests** (Producer/Consumer): Users can
+    now view all of their team's ACL requests for both producers and
+    consumers using the new UI.
+-   **My Team's Schema Requests**: Users can now view all of their
+    team's schema requests using the new UI.
+
+![image](../../static/images/release-220-react-ui.png)
+
+To preview the new Klaw user interface, open the
+`application.properties` file on the Klaw **core** module, and set the
+value of the following property to `true`: :: \# Enable new Klaw user
```

Comment:
The following should be on a new line I missed this the first time I'm sorry, but the note looks perfect now.
# Enable new Klaw user interface 
klaw.coral.enabled=true